site/docs/javadoc/index.html

```diff
@@ -1,9 +1,9 @@
 <html xmlns="http://www.w3.org/1999/xhtml">    
   <head>      
     <title>Iceberg Javadoc Redirect</title>      
-    <meta http-equiv="refresh" content="0;URL='/javadoc/0.10.0/'" />
+    <meta http-equiv="refresh" content="0;URL='/javadoc/0.11.0/'" />
   </head>    
   <body> 
-    <p>Redirecting to Javadoc for the 0.10.0 release: <a href="/javadoc/0.10.0/">/javadoc/0.10.0</a>.</p>
+    <p>Redirecting to Javadoc for the 0.11.0 release: <a href="/javadoc/0.11.0/">/javadoc/0.11.0</a>.</p>
```

Comment:
Not sure how javadoc generated, is there anything that has to be manually done to have the 0.11 javadoc folder?

Reply:
It's in `tasks.gradle`. Run `./gradlew refreshJavadoc`.

Reply:
Thanks, I added a note in README.